### PR TITLE
refactor(fe2): Changes to Move Projects button

### PR DESCRIPTION
**Pull request by andrewwallacespeckle**
- Moved the `Move projects` button down next to the `New projects` button and turned it into an actual button (white, similar to the invite button).
  - The button still becomes hidden in <md breakpoint, and instead is added to the ... menu
- Kept it visible to Members and Guests but disabled. On hover show tooltip with: "Only admins can move projects into the workspace".
  - Tooltips don't consistently work on buttons, so i wrapped the button in a div.
- Add "Move projects" button to Quickstart card

### CODE REVIEW

**Comment by benjaminvo** [comment]
Maybe too early for feedback since this is in draft but I see the tooltip even though I'm an admin.

CleanShot 2024-10-10 at 15 58 57@2x

**Comment by andrewwallacespeckle** [comment]
> Maybe too early for feedback since this is in draft but I see the tooltip even though I'm an admin.
> 
> CleanShot 2024-10-10 at 15 58 57@2x

This seems to be a reactivity bug. On reload, it works as expected. I'll look into it now.

File: packages/frontend-2/components/workspace/ProjectList.vue [hunk 1]
```diff
@@ -32,9 +32,30 @@
             v-bind="bind"
             v-on="on"
           />
-          <FormButton v-if="!isWorkspaceGuest" @click="openNewProject = true">
-            New project
-          </FormButton>
+          <div class="flex gap-2">
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/ProjectList.vue:35]: @Mikehrn It seems that v-tippy is not reactive. When you go from a workspace page that you are not an admin of, to one you are an admin, the buttons will correctly be enabled, but the tooltip remains.

Conditionally rendering the surrounding div seems to fix this, but open to suggestions if you have any other ideas.

File: packages/frontend-2/components/workspace/header/Header.vue [hunk 2]
```diff
@@ -189,12 +181,19 @@ const team = computed(() => props.workspaceInfo.team.items || [])
...
+const isWorkspaceGuest = computed(
+  () => props.workspaceInfo.role === Roles.Workspace.Guest
+)
 const actionsItems = computed<LayoutMenuItem[][]>(() => [
   [
     ...(isMobile.value
       ? [
-          { title: 'Move projects', id: ActionTypes.MoveProjects },
-          { title: 'Invite', id: ActionTypes.Invite }
+          ...(isWorkspaceAdmin.value
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/header/Header.vue:191]: Previously the mobile buttons where always shown, now they are hidden based on the same permissions as we use on desktop.

File: packages/frontend-2/components/workspace/ProjectList.vue [hunk 3]
```diff
@@ -32,9 +32,30 @@
...
+            <!--- Conditionally apply tooltip only for non-admins and avoid v-tippy reactivity bug -->
+            <div v-if="!isWorkspaceAdmin" v-tippy="'You must be a workspace admin'">
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/workspace/ProjectList.vue:37]: @benjaminvo This is different tooltip text to what you suggested in Linear. It seems our tooltips cant handle longer text right now. 

I've shortened it, but feel free to update this, or suggest something better. I can also look at improving the tooltips, and going with the original text if you think it is needed. 

image

**Comment by andrewwallacespeckle** [comment]
> > Maybe too early for feedback since this is in draft but I see the tooltip even though I'm an admin.
> > CleanShot 2024-10-10 at 15 58 57@2x
> 
> This seems to be a reactivity bug. On reload, it works as expected. I'll look into it now.

This has been fixed

File: packages/frontend-2/components/workspace/ProjectList.vue (hunk 3 above)
Comment (benjaminvo) [reply packages/frontend-2/components/workspace/ProjectList.vue:37]: I like it!

**Review by Mikehrn — APPROVED** [review]

**Pull request merged**